README.md

```diff
@@ -119,6 +120,12 @@ For titles there is no way it can be empty, at least the last fallback will alwa
 
 ## Changelog
 
+### Version 1.4.0
+
+* Added config option to return separate fallback image for frontpage.
+* Added config option to return raw (unscaled) fallback images for both frontpage and normal content.
+* Added config option to return fallback images as https.
```

Comment:
Remove this line too and I can merge for 1.4.0, hopefully tomorrow if I get the time to do some testing.

Reply:
Good catch! Done.